core/src/worker/activities/activity_heartbeat_manager.rs

```diff
@@ -353,6 +364,7 @@ impl ActivityHeartbeatManager {
                                 // finished (which is one thing not found implies - other reasons
                                 // would seem equally valid).
                                 Err(s) if s.code() == tonic::Code::NotFound => {
+                                    warn!("Act not found when recording heartbeat");
```

Comment:
Don't we warn about this somewhere else?
Also, if this is user visible, let's not shorten the word `Activity`.

Reply:
Both these I already fixed